src/main/java/ForkJoinSolution.java

```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.Future;
+
+public class ForkJoinSolution {
+    public int solution(List<Integer> list) {
+        List<Callable<Integer>> callables = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            callables.add(new MyCallable(list, i * 100_000, (i + 1) * 100_000));
+        }
+
+        ExecutorService forkJoin = new ForkJoinPool(10);
```

Comment:
```suggestion
        ExecutorService forkJoin = new ForkJoinPool(10);
```
In this class you should "Try doing all logic using recursion in method compute() of RecursiveTask." and use ForkJoinTask class insted ExecutorService